backend/src/check/check.service.ts

```
@@ -33,64 +33,91 @@ export class CheckService {
 				checkYorkieDto.method
 			)
 		) {
-			allowed = true;
-			reason = `Pass ${checkYorkieDto.method}`;
+			return {
+				allowed: true,
+				reason: `Pass ${checkYorkieDto.method} method`,
+			};
+		}
+
+		const { key: yorkieDocumentId } = checkYorkieDto.attributes?.[0];
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)